elf/elf.go

```diff
@@ -555,6 +555,7 @@ func (b *Module) Load(parameters map[string]SectionParams) error {
 			isTracepoint := strings.HasPrefix(secName, "tracepoint/")
 			isSchedCls := strings.HasPrefix(secName, "sched_cls/")
 			isSchedAct := strings.HasPrefix(secName, "sched_act/")
+			isSkSkb := strings.HasPrefix(secName, "sk/skb/")
```

Comment:
gobpf and libbpf unfortunately diverged in the way they name sections:
https://github.com/torvalds/linux/blob/v5.0/tools/lib/bpf/libbpf.c#L2610-L2662

But when possible, could we adopt the same section names as in libbpf? That would be:
- `sk_skb/stream_parser`
- `sk_skb/stream_verdict`